recipe/build.sh

```diff
@@ -1,23 +1,8 @@
 #!/usr/bin/env bash
-
-cd "${SRC_DIR}/.."
-mkdir "src"
-mkdir "src/github.com"
-mkdir "src/github.com/opencontainers"
-mkdir "src/github.com/tianon"
-mv "${SRC_DIR}" "src/github.com/tianon/gosu"
-
-export RUNC_VERSION="0.1.0"
-curl -L https://github.com/opencontainers/runc/archive/v${RUNC_VERSION}.tar.gz | tar -xzC "src/github.com/opencontainers"
-mv "src/github.com/opencontainers/runc-${RUNC_VERSION}" "src/github.com/opencontainers/runc"
-
 export CGO_ENABLED=0
 export GOARCH="amd64"
 export GOOS="linux"
```

Comment:
GOARCH and GOOS are set by the `go_{linux,osx,win}-64` packages if you are using conda's `target_platform`